examples/CMakeLists.txt

```diff
@@ -34,8 +34,10 @@ endif()
 # Alpaka examples #
 ###################
 
-set(ALPAKA_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../../alpaka/" CACHE STRING  "The location of the alpaka library")
-LIST(APPEND CMAKE_PREFIX_PATH "${ALPAKA_ROOT}")
+set(ALPAKA_ROOT "ALPAKA-NOTFOUND" CACHE STRING  "The location of the alpaka library")
+if(NOT ALPAKA_ROOT STREQUAL "ALPAKA-NOTFOUND")
```

Comment:
I would say this must be `if(ALPAKA_ROOT)` only. I  will check it.

Reply:
Thanks, it took me 5 minutes, to figure out how to compare strings in CMake!

Reply:
but... I linked it [above](https://github.com/ComputationalRadiationPhysics/llama/pull/19#issuecomment-474755430) :)